src/tsolvers/lasolver/Tableau.cpp

```diff
@@ -18,38 +18,24 @@ namespace {
 }
 
 void Tableau::nonbasicVar(LVRef v) {
-    if(contains(nonbasic_vars, v)) {return;}
-    assert(!contains(basic_vars, v));
+    if(isNonBasic(v)) {return;}
+    assert(!isProcessed(v));
     newNonbasicVar(v);
 }
 
 void Tableau::newNonbasicVar(LVRef v) {
-    assert(!contains(nonbasic_vars, v));
-    while(cols.size() <= v.x) {
-        cols.emplace_back();
-    }
-    while(rows.size() <= v.x) {
-        rows.emplace_back();
-    }
+    assert(!isProcessed(v));
+    ensureTableaReadyFor(v);
```

Comment:
Is this a typo, should it be ensureTableauReadyFor(v)?

Reply:
Typo -> fixed